c_glib/meson.build

```diff
@@ -23,7 +23,8 @@ project('arrow-glib', 'c', 'cpp',
           'cpp_std=c++11',
         ])
 
-version = run_command('python', 'tool/get-version.py').stdout().strip()
+python = find_program('python3', 'python', 'python2')
```

Comment:
Is the order here also the preference used? Normally `python` is the preferred one when you're in `virtualenv` or a `conda` env. Thus as a Python developer, I would like always the plain `python` executable one as the preferred.

Reply:
OK. Let's use `'python', 'python3', 'python2'` order.

Reply:
I've changed the order.